### ISSUE TITLE
iOS 14+'s built-in animated WebP support (via SDWebImageAWebPCoder) have bad performance on large image

### ISSUE THREAD

**Issue by dreampiggy** [SDWebImage/SDWebImage#3132]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | ios
 Platform Version        | 14.0
 SDWebImage Version      | 5.9.0+
 Integration Method      | cocoapods
 Xcode Version           | Xcode 12
 Repro rate              | all the time (100%)
 Repro with our demo prj | e.g. does it happen with our demo project? Yes.
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

When use `SDWebImageAWebPCoder` introduced from SDWebImage 5.9.0, I found that it contains bad performance, compared to previous `SDWebImageWebPCoder`.

Test Image:

https://p1-ppx.byteimg.com/img/tos-cn-i-0000/424e830d3b5447b8802b2ebd900f90e3~tpl-1_q30_492x480.webp

Test Result:

[Compare Result.mp4.zip](https://github.com/SDWebImage/SDWebImage/files/5595347/Compare.Result.mp4.zip)

I guess this is Apple's bug, so I also submit the radar for this:

![image](https://user-images.githubusercontent.com/6919743/100202137-e1096f80-2f3b-11eb-8678-80edbb5df055.png)

**Comment by wulongwang** [comment]
I have the same problem
for example 
https://pic.rmb.bdstatic.com/2021-1/1610332479407/37d4f305c2db.webp

**Comment by dreampiggy** [comment]
@wulongwang  Use SDWebImageWebPCoder if you care about Animated WebP performance.

**Comment by austinfitzpatrick** [comment]
I know this is an old issue, but I'm also seeing this.  SDImageWebPCoder is much slower to parse than SDImageAWebPCoder but once I set the resulting UIImage onto a UIImageView the performance hit is noticeable.  I think the UIImageView is decoding it on the main thread when I set the Image, instead of before hand.  Maybe there's a way to pre-load these on a background thread?

Scrolling performance on a list of Animated WebPs is bad with this new coder.
